src/tests/unit/findWithPagination.test.ts

```diff
@@ -90,12 +116,11 @@ describe("pagination", () => {
       where: { foo: Like("page-test-%") },
       join: { alias: "f", innerJoin: { o: "f.owner" } },
       order: { "o.name": "DESC", foo: "DESC" },
-      pagination: { first: 2 },
+      pagination: { first: 3 },
```

Comment:
I increased the pagination count request to 3 per page in order to add assertions on both of the orders in the request (ie, test that the results are sorted first by name DESC but then also by foo DESC